bin/steps/conda_compile

```diff
@@ -23,16 +24,21 @@ if [ ! -d /app/.heroku/miniconda ]; then
         cp .condarc /app/.heroku/miniconda
     fi
 
+    echo "nomkl" > $HOME/.heroku/miniconda/conda-meta/pinned
+    echo "python <3.8.0" >> $HOME/.heroku/miniconda/conda-meta/pinned
+    # Pin the conda version if proper version, not `latest` or anything else
+    if [[ $VERSION =~ ^[0-9]+(\.[0-9]+)*$ ]]; then
+        echo "conda ==${VERSION}" >> $HOME/.heroku/miniconda/conda-meta/pinned
+    fi
+    echo "added pinned file in $HOME/.heroku/miniconda/conda-meta/pinned"
+
     conda config --set auto_update_conda False
     conda install $MINICONDA_VERBOSITY --no-update-dependencies pip --yes | indent
 fi
 
-echo "nomkl" > $HOME/.heroku/miniconda/conda-meta/pinned
-# Pin the conda version if proper version, not `latest` or anything else
-if [[ $VERSION =~ ^[0-9]+(\.[0-9]+)*$ ]]; then
-	echo "conda ==${VERSION}" >> $HOME/.heroku/miniconda/conda-meta/pinned
-fi
-echo "added pinned file in $HOME/.heroku/miniconda/conda-meta/pinned"
+# echo "Sleeping for 15 minutes..."
```

Comment:
This should be removed (via a rebase or amend).

Reply:
Oh, my bad, will rebase. Thanks